src/metrics_test.go

```diff
@@ -234,8 +237,8 @@ func TestPopulateMetrics(t *testing.T) {
 	}
 
 	// TODO: use assertions library for tests
-	if len(metricsSet) != 13 {
-		t.Errorf("metricsSet length = %d. Expected 13", len(metricsSet))
+	if len(metricsSet) != 17 {
```

Comment:
magic number 😲 , since it's a test I would put a message where is this number coming from, wdyt?